src/standard/effectBuilder.html

```diff
@@ -221,6 +221,7 @@
           a.literal = true;
           break;
         case '#':
+        case '-':
```

Comment:
I think we should check this far earlier, and then check that the next character is a number literal.

Something like

```js
var fc = arg[0];
if (fc === '-') {
  fc = arg[1];
}
if (fc >= '0' && fc <= '9') {
```